reference-architectures/cloud_deploy_flow/CloudRun/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.23 as builder
```

Comment:
Consider adding a GitHub Actions workflow file that builds this container image. Maybe in a later PR because this is already quite big.

Reply:
Another PR, I should learn more about that process